### ISSUE TITLE
Pretrain code?

### ISSUE THREAD

**Issue by iamSTone** [slim1017/VaDE#2]
Where can I see the codes to make files such as 'ae_XXXX.json' and ''ae_XXXX_weights.h5'?

**Comment by Jossome** [comment]
I kind of suspect that the pre-trained weights are gained through a supervised manner. I removed the pretrain step of clsutering on MNIST, and the accuracy stuck at 0.11, where all data are assigned to the same class label. Yet the loss keeps going down, so I put "kind of" suspect. Maybe it's because I didn't go thru enough epochs, but a 0.11 accuracy in the beginning is not convincing at all.
